From 971769f63ec25cea7b64a4a734d257cff4b63ba9 Mon Sep 17 00:00:00 2001
From: wangjiezhe <wangjiezhe@gmail.com>
Date: Tue, 2 May 2023 20:27:09 +0800
Subject: [PATCH] Honor PATH elements set by WSL

Signed-off-by: wangjiezhe <wangjiezhe@gmail.com>
---
 etc/profile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/etc/profile b/etc/profile
index 2be4622a..84e5c31d 100644
--- a/etc/profile
+++ b/etc/profile
@@ -6,8 +6,11 @@
 
 # Load environment settings from profile.env, which is created by
 # env-update from the files in /etc/env.d
+# Store Windows paths first
 if [ -e /etc/profile.env ] ; then
+	[ -z "${WINPATH}" ] && WINPATH=${PATH}
 	. /etc/profile.env
+	PATH="${PATH}:${WINPATH}"
 fi
 
 # You should override these in your ~/.bashrc (or equivalent) for per-user
-- 
2.39.2

